### DISCUSSION TITLE
import/export cannot support translations

### DISCUSSION THREAD

**Discussion by bwisely** [k-ivan/visual-bookmarks-chrome#174] [General]
Hi Ivan,
Thank you for updating the language to support the Taiwan Chinese language.
And I found some issues,
1. tab title for the VB, can we remove some text? The currently shown text is => Visual bookmarks(視覺書籤); it's too long.
2. The import/export is still in English, not Chinese string.
3. I don't know what is "message" is: "Include a panel with services"

Wisely

**Comment by k-ivan** [comment]
Hi @bwisely

Thank you for your help!

1. Let's keep only the Chinese version.
2. My mistake, I didn't include them in the translations. I'll correct this in the coming days.
3. The service panel is essentially the same as links to websites, but instead of using them as bookmarks, we want quick access to them.(below, I’ve provided a screenshot showing what they look like). That's why I called it services.

![image](https://github.com/user-attachments/assets/5a9dbf59-e5b0-44ad-a327-3e1c18913c02)

**Reply by bwisely** [reply]
My mistake; I didn't include them in the translations. I'll correct this in the coming days.  ==> You can see the captured as below
![Snap2](https://github.com/user-attachments/assets/2c772c0a-4596-4580-b194-afc5d77f8bb7)

The service panel is essentially the same as links to websites, but instead of using them as bookmarks, we want quick access to them.(below, I’ve provided a screenshot showing what they look like). That's why I called it services.
==> So, I think we need to write more description for this.

**Reply by k-ivan** [reply]
> So, I think we need to write more description for this.

I'll think about the description.

**Comment by k-ivan** [comment]
Hi @bwisely

Fixed all the points, added more description to the 3rd one. (AI translation). v6.2.2
